apollo-router/src/plugins/telemetry/tracing/datadog.rs

```diff
@@ -79,6 +92,19 @@ pub(crate) struct Config {
     /// http_request -> http.route
     #[serde(default)]
     resource_mapping: HashMap<String, String>,
+
+    /// Which spans will be eligable for span stats to be collected for viewing in the APM view.
+    /// Defaults to true for `router` and `supergraph`, `subgraph`, `subgraph_request`, `http_request` and `query_planning`.
```

Comment:
This comment isn't accurate. It is missing at least execution. I would expect the defaults to be as listed in `BUILT_IN_SPAN_NAMES`, so probably better to just reference that.

Reply:
Fixed in 3ee8b3a. The reason not to reference `BUILT_IN_SPAN_NAMES` is that the docs are exposed to the user for via the json schema.